pkg/build/fuchsia.go

```diff
@@ -92,3 +104,21 @@ func runSandboxed(timeout time.Duration, dir, command string, arg ...string) ([]
 	}
 	return osutil.Run(timeout, cmd)
 }
+
+func genSSHKeys(dir string) (string, string, error) {
+	// Fuchsia images no longer include ssh keys. Create an ssh key and append the public key to the zbi.
```

Comment:
Nit: This belongs up with the call to `zbi` and not here (Also, I think the "no longer" sentence isn't necessary historical context at this point.)